### DISCUSSION TITLE
No precompiled xaml found when trying to include Theme

### DISCUSSION THREAD

**Discussion by T0biasCZe** [AvaloniaUI/Avalonia#15881] [Q&A]
I want to apply the [Avalonia Aero](https://github.com/Splitwirez/avalonia-aero-theme/tree/migration/avalonia--11-0-5) theme to my project, I added the AvaloniaAero.csproj as existing project into my solution, but when I try to include it in App.axaml, it throws error that no precompiled xaml was found
`Avalonia.Markup.Xaml.XamlLoadException: 'No precompiled XAML found for avares://AvaloniaAero/AeroTheme.axaml (baseUri: avares://DSFotoToolDeluxe/App.axaml), make sure to specify x:Class and include your XAML file as AvaloniaResource'`

this is the code
```
<Application xmlns="https://github.com/avaloniaui"
             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
             x:Class="DSFotoToolDeluxe.App"
             RequestedThemeVariant="Default">
  <!-- "Default" ThemeVariant follows system theme variant. "Dark" or "Light" are other available options. -->

  <Application.Styles>
    <StyleInclude Source="avares://AvaloniaAero/AeroTheme.axaml"/>
  </Application.Styles>
</Application>
```

**Comment by stevemonaco** [comment]
Try adding: `xmlns:aero="using:AvaloniaAero"`
And using: `<aero:AeroTheme />` instead of the `StyleInclude`.

Make sure you're using the migration branch and hope it works. Otherwise, open an issue in their repo. You need to be very cautious when using libraries that you aren't sure has been successfully ported to Avalonia 11 and that involves reading the library's source code, at least in part. The main branch is still on an early 11 preview version.

**Comment by T0biasCZe** [comment]
I am using the migration branch (avalonia--11-0-5), and i tried the code you suggested and it throws error immediately during build
```
Severity	Code	Description	Project	File	Line	Suppression State	Details
Error	AXN0002	XamlX.XamlParseException: Unable to resolve type AeroTheme from namespace using:AvaloniaAero Line 9, position 6.	DSFotoToolDeluxe	C:\Users\user\source\repos\DSFotoToolDeluxe\DSFotoToolDeluxe\CSC	1		

```

**Comment by stevemonaco** [comment]
I'd suggest you fork the project, read through their demo, and run it.

Otherwise, I'm not sure what is wrong. You may have included their source incorrectly.
